simplicity-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "simplicity-sys"
```

Comment:
should we change also this in `simplicity-lang-sys` ?

Reply:
I like `simplicity-sys` because it is shorter. There will be a lot of projects with the `simplicity` infix.

We invade the namespace of the existing `simplicity` project on crates.io, but the last update was 3 years ago so it seems pretty dead.